runtime-modules/storage/src/tests/mock.rs

```diff
@@ -401,6 +402,7 @@ pub(crate) fn hire_storage_provider() -> (u64, u32) {
         role_account_id,
         reward_relationship: None,
         role_stake_profile: None,
+        storage: Vec::new(),
```

Comment:
one more to remove..

Reply:
Done https://github.com/Joystream/joystream/pull/2180/commits/dc39181954b99319c9fb98b0ff9cecb3ec475fe5